config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
     devise_for :users
-    resources :books
+    resources :books do
```

Comment:
Here, it's recommended to use `only` to define which routes do you want to generate of the next ones: `index`, `create`, `show`, `update` and `destroy`... 

In your case it would be: 
```
resources :books, only: [index, show, create] do
    resources :comments
end
```
This way you avoid to generate unnecessary routes as update and delete that you are not using right now